app/src/main/java/co/condorlabs/customcomponents/imagecropview/AppCompatCropImageView.java

```diff
@@ -142,6 +140,12 @@ protected void onDraw(Canvas canvas) {
                 resizeDrawable2.draw(canvas);
                 resizeDrawable3.draw(canvas);
             }
+            // Calculate initial aspectRatio
```

Comment:
should create a method called calculateInitialAspectRatio

Reply:
Ok